lambda_function/src/vars.yaml

```diff
@@ -0,0 +1,13 @@
+# Mission name
```

Comment:
can this file be called something more descriptive like config.yaml. vars could be anything.